tf-modules/log-subscription-filter/ingest_log_subscriptions.tf

```diff
@@ -0,0 +1,44 @@
+resource "aws_cloudwatch_log_subscription_filter" "discover_pdrs_task_log_subscription_filter" {
+  count           = var.log_destination_arn != null && var.ingest_logs_to_elk ? 1 : 0
```

Comment:
I don't know terraform, I assume a bool true will convert to 1 and false to 0?

Reply:
Yes. Works just like the ternary operator in other languages.